src/js/mixins/DuplicableRowsMixin.jsx

```diff
@@ -40,7 +40,7 @@ var DuplicableRowsMixin = {
     Object.keys(duplicableRowsScheme).forEach(function (fieldId) {
       if (state.rows[fieldId] == null || state.rows[fieldId].length === 0) {
         let rowScheme = Util.extendObject(duplicableRowsScheme[fieldId], {
-          consecutiveKey: Util.getUniqueId()
+          consecutiveKey: parseInt((Util.getUniqueId() + "").slice(-9))
```

Comment:
There is already a fix somewhere else, using an object instead of an array.

Not sure where this fix is, will find it out.